### PR TITLE
libdrm changes to support rpl-s platform

### PR DESCRIPTION
**Pull request by bkothapx**
Tracked-On: OAM-105280
Signed-off-by: kalle <kalyan.alle@intel.com>
Signed-off-by: Kothapeta, BikshapathiX <bikshapathix.kothapeta@intel.com>

### CODE REVIEW

**Comment by bkothapx** [comment]
Reviewers: @hongyu.zhang@intel.com  @yu.y.chen@intel.com @hongcheng.xie@intel.com  @jeevaka.badrappan@intel.com; @qiming.shi@intel.com @shyjumon.n@intel.com

**Comment by bkothapx** [comment]
Reviewers: @zhy1030 @hongcheng-xie @JeevakaPrabu @shyjumon-n

**Review by feijiang1 — APPROVED** [review]

File: intel/intel_chipset.c [hunk 1]
```diff
@@ -35,6 +35,7 @@ static const struct pci_device {
 	uint16_t gen;
 } pciids[] = {
 	/* Keep ids sorted by gen; latest gen first */
+	INTEL_RPLS_IDS(12),
```
Comment (JeevakaPrabu) [inline intel/intel_chipset.c:38]: RPL is 13th generation. Could you please check and correct the generation information mapping for all platforms?

**Review by JeevakaPrabu — CHANGES_REQUESTED** [review]

**Review by JeevakaPrabu — APPROVED** [review]

**Review by feijiang1 — APPROVED** [review]

**Review by shyjumon-n — APPROVED** [review]
LGTM

**Review by shyjumon-n — APPROVED** [review]
LGTM

**Review by feijiang1 — APPROVED** [review]

**Pull request merged**